pkg/resources/jobs/runner.go

```diff
@@ -72,7 +72,7 @@ func NewRunnerJob(k *v1alpha1.K6, index int) (*batchv1.Job, error) {
 						Ports: ports,
 					}},
 					TerminationGracePeriodSeconds: &zero,
-					Volumes:                       newVolumeSpec(k.Spec.Script),
+					Volumes:                       newVolumeSpec(k.Spec.Script, k.Spec.VolumeClaim),
```

Comment:
Maybe it would be preferable to pass the whole Spec instead of adding an additional param?